lenovo/legion/16aph8/default.nix

```
@@ -0,0 +1,39 @@
+{ lib, pkgs, ... }:
+
+{
+  imports = [
+    ../../../common/cpu/amd
+    ../../../common/cpu/amd/pstate.nix
+    ../../../common/gpu/amd
+    ../../../common/gpu/nvidia
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/ssd
+  ];
+
+  # Use latest LTS kernel for more Raphael fixes
+  boot = lib.mkMerge [
+    (lib.mkIf (lib.versionOlder pkgs.linux.version "6.6") {
+      kernelPackages = pkgs.linuxPackages_6_6;
```

We don't want to downgrade the kernel in future:

```suggestion
      kernelPackages = pkgs.linuxPackages_latest;
```